CMakeUtilities/InstallCMakeExports.cmake

```diff
@@ -111,17 +113,36 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/FindSIMDIS_SDK.cmake
     DESTINATION ${SDK_LIB_CMAKE_PATH}
     COMPONENT Exports
 )
+endif()
 
-# Configure and install the ImportSIMDIS_SDK.cmake useful for external projects
-configure_file(CMakeUtilities/ImportSIMDIS_SDK.cmake.in
-    ${CMAKE_CURRENT_BINARY_DIR}/cmake/ImportSIMDIS_SDK.cmake
-    @ONLY
-)
-install(FILES
+if(USING_VCPKG)
+    # Configure and install the ImportSIMDIS_SDK.cmake useful for external projects
+    configure_file(CMakeUtilities/FindSIMDIS_SDK.cmake.in
         ${CMAKE_CURRENT_BINARY_DIR}/cmake/ImportSIMDIS_SDK.cmake
```

Comment:
`FindSIMDIS_SDK.cmake.in` definitely should not be creating an `ImportSIMDIS_SDK.cmake` output file.